### PR TITLE
[explorer] Tweak to show transactions count

### PR DESCRIPTION
**Pull request by Jordan-Mysten**
## Description 

Reworks transaction block table to remove amounts and show transaction count.

## Test Plan 

Ran locally.

### CODE REVIEW

**Comment by Jordan-Mysten** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10054** <a href="https://app.graphite.dev/github/pr/MystenLabs/sui/10054" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/MystenLabs/sui/10054?utm_source=stack-comment).

File: apps/explorer/src/components/transactions/TxCardUtils.tsx [hunk 1]
```diff
@@ -105,9 +97,13 @@ export const genTableDataFromTxData = (
                     />
                 </TxTableCol>
             ),
-            amounts: (
+            txns: (
                 <TxTableCol>
-                    <SuiAmount amount={suiAmount} />
+                    {transaction.transaction?.data.transaction.kind ===
+                    'ProgrammableTransaction'
+                        ? transaction.transaction.data.transaction.transactions
```
Comment (williamrobertson13) [inline apps/explorer/src/components/transactions/TxCardUtils.tsx:104]: `transaction.transaction.data.transaction.transactions` lol 🤔

**Review by williamrobertson13 — APPROVED** [review]
lgtm

File: apps/explorer/src/components/transactions/TxCardUtils.tsx (hunk 1 above)
Comment (Jordan-Mysten) [reply apps/explorer/src/components/transactions/TxCardUtils.tsx:104]: Haha yeah I realized how much we need to rename some of these APIs now...

**Pull request merged**